src/bap_mc/bap_mc.ml

```diff
@@ -15,10 +15,10 @@ exception Unknown_arch
 exception Can't_lift of Error.t
 exception Trailing_data of int
 
-let no_disassembly state boff =
+let no_disassembly state boff offset =
```

Comment:
I thought that this was a good side step to the compiler errors that I mention in other comment. So this issue arose because of other issues. It's clearly not right, I will fix it.